Verbose/English/Help.lean

```diff
@@ -790,9 +790,8 @@ example (P : Prop) (h : P) : True := by
 -- TODO: Improve this help message (low priority since it is very rare)
 /--
 info: Help
-• (
-  Let's prove it's contradictory
-  We conclude by h)
+• ( Let's prove it's contradictory
```

Comment:
These are probably fall-out from Lean core changes (but look neutral or like an improvement to me). Same for the French ones.